draggablepanel/src/main/java/com/github/pedrovgs/DraggableListener.java

```diff
@@ -42,4 +42,28 @@
    * Called when the view is closed to the right.
    */
   void onClosedToRight();
+
+  /**
+   * Called when the view is being touched at present
+   */
+
+  void onTouchListener();
+
+  /**
+   * Called when the view is maximized with click instead of drag
+   */
+
+  void clickedToMaximize();
+
+  /**
+   * Called when the view is minimized with click instead of drag
+   */
+
+  void clickedToMinimize();
+
+  /**
+   * Called when the view is being dragged to either minimize or maximize
+   */
+
+  void smoothSlide();
```

Comment:
We should change the naming of the methods you've added to match with the naming of the project. If you review the methods already implemented all start using ``on`` as prefix. Based on that, we should rename these last three methods to:

```java
void onTouch()

void onClickedToMaximize()

void onClickedToMinimize();

void onSmoothSlide();
```